editions/tw5.com/tiddlers/concepts/WikiText.tid

```diff
@@ -1,15 +1,18 @@
 created: 20131205155227468
-modified: 20140919191220377
+modified: 20220728185721771
 tags: Concepts Reference
 title: WikiText
 type: text/vnd.tiddlywiki
+user-story: [[Headings in WikiText]] [[Paragraphs in WikiText]] [[Hard Linebreaks in WikiText]] [[Formatting in WikiText]] [[Dashes in WikiText]] [[Block Quotes in WikiText]] [[Lists in WikiText]] [[Definitions in WikiText]] [[Horizontal Rules in WikiText]] [[Tables in WikiText]] [[Code Blocks in WikiText]] [[Typed Blocks in WikiText]] [[Images in WikiText]] [[Linking in WikiText]] [[Variables in WikiText]] [[Macro Definitions in WikiText]] [[Macro Calls in WikiText]] [[Macros in WikiText]] [[Transclusion in WikiText]] [[Transclusion and Substitution]] [[Styles and Classes in WikiText]] [[HTML in WikiText]] [[Widgets in WikiText]] [[WikiText Parser Modes]]
```

Comment:
It looks like the demo works by modifying the docs tiddler itself; that's bad because it triggers the dirty state (we had the same problem recently with the GettingStarted tiddler). It would be better to use a temporary tiddler, just using this field to copy from if the field is missing or blank.

Reply:
Alternatively, perhaps it is not necessary to make the ordering editable? In any event, I don't think that ordering the docs is sufficient; we also need to introduce group headings or something to show the gross structure.